app/scripts/modules/core/help/helpField.directive.js

```diff
@@ -50,6 +51,22 @@ module.exports = angular
           scope.$watch('key', applyContents);
           scope.$watch('fallback', applyContents);
           scope.$watch('content', applyContents);
+
+          let tooltipShownStart = null;
```

Comment:
Agreed! I spent a good 20 minutes looking around for some event hooks before accepting they don't exist.

Then again, this is the first time I've ever had a use for those kind of hooks, so not sure if other folks would want them...